common/beerocks/bcl/include/beerocks/bcl/network/net_struct.h

```diff
@@ -22,17 +22,19 @@ inline bool operator==(sMacAddr const &lhs, sMacAddr const &rhs)
 
 inline bool operator!=(sMacAddr const &lhs, sMacAddr const &rhs) { return !(rhs == lhs); }
 
-template <> struct std::hash<sMacAddr> {
+namespace std {
+template <> struct hash<sMacAddr> {
     size_t operator()(const sMacAddr &m) const
     {
         uint64_t value_to_hash = 0;
         for (size_t byte = 0; byte < sizeof(m.oct); byte++) {
             value_to_hash <<= 8;
             value_to_hash += m.oct[byte];
         }
-        return std::hash<std::uint64_t>()(value_to_hash);
+        return hash<std::uint64_t>()(value_to_hash);
     }
 };
+}
```

Comment:
There should be a comment here that marks the end of the namespace. `clang-format.sh` adds it.

I'll rebase on master, add that comment, and push it.